custom_components/waste_collection_schedule/waste_collection_schedule/source/ccc_govt_nz.py

```diff
@@ -1,77 +1,78 @@
 import datetime
 
-import requests
 from waste_collection_schedule import Collection
-
-# Include work around for SSL UNSAFE_LEGACY_RENEGOTIATION_DISABLED error
 from waste_collection_schedule.service.SSLError import get_legacy_session
 
 
 TITLE = "Christchurch City Council"
-DESCRIPTION = "Source for Christchurch City Council."
-URL = "https://ccc.govt.nz"
-TEST_CASES = {"53 Hereford Street": {"address": "53 Hereford Street"}}
+DESCRIPTION = "Source for Christchurch City Council"
+URL = "https://ccc.govt.nz/services/rubbish-and-recycling/collections"
```

Comment:
The URL should only be the Domain not a full URL as it is shown in the README.md